controllers/logging.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * logging.js
+ * Callbacks for logging events to the database
+ */
+
+const db = require('../db/connect');
+
+function logConnection(payload) {
+
+    console.log("Logging: logConnection fired");    
+
+    // Instantiate new chat document
+    const chat = new db.models.Chat({
```

Comment:
you should write `const Chat = db.models.Chat` earlier in the file so you don't have to reference `db.models.Chat` throughout the file